docker/Dockerfile

```diff
@@ -71,9 +77,14 @@ RUN --mount=type=cache,target=./target,uid=10001,gid=10001 set -ex; \
         cargo clean --release; \
         cargo build --features "${CARGO_BUILD_FEATURES}" --release; \
         mv "./target/$CARGO_BUILD_TARGET/$RUST_RELEASE_MODE/lemmy_server" /home/lemmy/lemmy_server; \
+        #
+        # Compress the binary with upx
+        wget https://github.com/upx/upx/releases/download/v4.2.4/upx-4.2.4-arm64_linux.tar.xz; \
+        tar -xvf upx-4.2.4-arm64_linux.tar.xz; \
+        cp upx-4.2.4-arm64_linux/upx /usr/bin; \
+        upx --best --lzma /home/lemmy/lemmy_server; \
     fi
```

Comment:
No need to compress in debug mode.

Reply:
This is the release if block, the debug one is on L68-L72

Reply:
Oh this is for ARM releases and the other one for x86, alright then.